setup-wallets-programmatically.html

```diff
@@ -104,124 +104,63 @@
     <div class="content doc-content">
```

Comment:
Because the Markdown file has changed, rename the output HTML file and rebuild and recommit, so that this file disappears.